### ISSUE TITLE
runmatch.py file removed?

### ISSUE THREAD

**Issue by jdebacker** [PSLmodels/taxdata#403]
This documentation (`Match.md`) references a `runmatch.py` file, but I can't find this in the repo.  Has it been removed?  I see a call to the file in `Makefile` has been commented out.

**Comment by andersonfrailey** [comment]
Yes, `runmatch.py` was removed with the refactor in #367. I'll go through the documentation and update it accordingly.
